misk/src/main/kotlin/misk/cloud/fake/security/keys/FakeKeyManager.kt

```diff
@@ -0,0 +1,33 @@
+package misk.cloud.fake.security.keys
+
+import misk.security.keys.KeyManager
+import okio.ByteString
+import java.nio.ByteBuffer
+import java.security.Key
+import javax.crypto.Cipher
+import javax.crypto.spec.SecretKeySpec
+
+internal class FakeKeyManager : KeyManager {
+    override fun encrypt(keyAlias: String, plainText: ByteString): ByteString {
+        val key = newSymmetricKey(keyAlias)
+        val cipher = Cipher.getInstance("AES")
```

Comment:
Maybe pull "AES" out in to a companion object val?

Reply:
I think probably not worth the indirection.